src/storyMap/components/StoryMapForm/configContext.js

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
@paulschreiber It will be the storyMap config context used by the StoryMapForm and some of its subcomponents, this help us to avoid passing the config through properties